package.json

```diff
@@ -7,15 +7,15 @@
   },
   "scripts": {
     "build": "npm run build-glean && rollup --config",
-    "build-storybook": "cross-env NODE_OPTIONS=--openssl-legacy-provider storybook build --static-dir ./public --config-dir .storybook",
+    "build-storybook": "cross-env NODE_OPTIONS=--openssl-legacy-provider storybook build --config-dir .storybook",
```

Comment:
Is `static-dir ./public` now the default? Is the parameter completely gone?

Reply:
The parameter is completely gone, the values are now provided via the `staticDirs` export in `main.js`